packages/react-components/src/icons/PreviousPage_Inactive.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it intentional to keep this as an svg?

Reply:
oops, will delete